lms/static/js/dashboard/views/course_list_view.js

```diff
@@ -0,0 +1,40 @@
+;(function (define) {
+    'use strict';
+
+    define(['jquery', 'common/js/components/views/list'], function ($, ListView) {
+
+        var CourseListView = ListView.extend({
+
+            initialize: function (options) {
+                this.itemViewClass = options.itemViewClass || this.itemViewClass;
+                this.template = options.template;
+                this.parent = options.parent;
+
+                this.itemViews = [];
+            },
+
+            render: function () {
+                var self = this;
+
+                this.$el.html($(this.template).html());
+                this.$('.course').each(function () {
```

Comment:
So with this approach you are rendering all of the DOM elements for the courses, and then scrolling through the DOM and attaching views to each? If so I would recommend a slightly more performant approach whereby you don't attach the course DOM elements first, you just loop through the model and create the views, appending them to the parent container.